Mods/Pet_Default_v5_Dark.theme

```diff
@@ -1,20 +1,24 @@
 @description Default v5 Dark Theme
-@version 1.0
+@version 1.1
 @changelog
   - Updated to v5.22 (added preview)
+v1.1
+  - MIDI Editor Colormap reworked
+  - Changes in the ME Tracklist
+  - Separators added  
 @provides
   Default_5_Dark.ReaperThemeZip http://stash.reaper.fm/29513/Default_5_Dark.ReaperThemeZip
```

Comment:
Firstly, thanks for the help how this has to be formatted, I still have to learn a lot.
With the theme I'm a little unsure, I was generally thinking about keeping the v1.0 as a dark, but pristine clone of the default theme in ReaPack and keep it like it is and it may be better to make a second theme for further changes like more layouts e.g.  a "v5 Dark Extended" or "v5 Dark2".
My main intention still is - with newcomers in mind - that it is possible to switch between the bright original and my mod back and forth without losing any settings or chosen layouts.
But with the separators I break my own rule...
ATM it's (as yet) the only theme in the ReaPack and I by the way feel honored that you asked me, so I feel some responsibility not to do weird things with it.
So would you in consideration of my explanations suggest to keep the v1.0 as it is (including the URL) and upload a v2.0 to the stash? Should the v2.0 be part of the ReaPack too?

Thanks a lot for your support!